### PR TITLE
Create 2017-11-27-overcoming-language-learning-frustrations-guest-pos…

### PR DESCRIPTION
**Pull request by LinguaPony**
…t-by-lingua-pony.md

### CODE REVIEW

File: _posts/2017-11-27-overcoming-language-learning-frustrations-guest-post-by-lingua-pony.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+But there is yet another and especially disturbing reason of language learning frustrations – and that’s meanness of people around us. 
+Sometimes it’s a teacher freaking out and saying all sorts of nasty things to a student for no reason. Sounds fantastic? Well, it does 
+happen; it has happened. The only way to deal with such a teacher is to fire him or her at once, but still it can be too late because 
+harm has been done. Also, if you seek feedback at online forums, make sure you pick a friendly, supportive community, in which your 
+fellow members will be aiming at helping you, as opposed to building up their self-esteem by means of destroying yours. Evil things 
+do happen online, unfortunately, since being on the web gives many a sense of impunity; once get burned by such an experience, some of 
+us hurt for months, if not for years. 
+
+Should we meet a person whose self-confidence has been harmed by someone else’s nastiness, it’s our duty to offer moral support. 
+Language learning community should be a brotherhood – not a battlefield. Sometimes it’s enough just to tell an insecure person that
```
Comment (emk) [inline _posts/2017-11-27-overcoming-language-learning-frustrations-guest-post-by-lingua-pony.md]: Let's not forget the sisters, too, who are too often forgotten! :-) 

(I actually think that female language learners might be the majority, but the guys tend to get more attention.)

File: _posts/2017-11-27-overcoming-language-learning-frustrations-guest-post-by-lingua-pony.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+word spoken sincerely at the right moment can really work miracles.
+
+To sum up, language learning frustrations usually have to do with the confidence of the learner. As long as the person in question 
+has the strength to walk the road – step after step, no matter how small – there will be progress, but fatigue, doubt and loss of 
+motivation can ruin it all. So, in order to succeed we need to learn to replenish our energy, to avoid overstraining ourselves and, 
+most importantly, to believe in ourselves. We need to learn to seek help from the right people and to run a mile from those whose 
+influence can be destructive. And, most importantly, we need to know exactly what we are doing it all for.
+
+> This guest post is written by forum member LinguaPony who speaks and writes in her native Russian and in English with equal easy 
+and is now learning Italian and Mandarin Chinese with four more languages to be added within three years. YOu can read her language
```
Comment (emk) [inline _posts/2017-11-27-overcoming-language-learning-frustrations-guest-post-by-lingua-pony.md]: "YOu" → "You"

**Review by emk — CHANGES_REQUESTED** [review]
Thank you so much for submitting a guest post; I enjoyed reading it greatly!

I've made a couple of very minor suggestions below, but they're really minor niggles. The overall essay is very well-written.

**Comment by LinguaPony** [comment]
Hi Eric,

Thanks for the kind words. I have made the necessary changes and committed
them.

Irina

**Review by emk — APPROVED** [review]

**Comment by emk** [comment]
OK, I've merged it! With luck, it should show up on the blog fairly soon.

**Comment by LinguaPony** [comment]
Thanks, it's there alright. :)

**Pull request merged**